sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -2026,4 +2026,69 @@ abstract class SessionCatalogSuite extends AnalysisTest with Eventually {
     assert(partitionSchema.fieldNames.toSeq == Seq("year", "month"))
     assert(partitionSchema.fields.length == 2)
   }
+
+  test("corrupted view metadata: mismatch between viewQueryColumnNames and schema") {
+    withSQLConf("spark.sql.viewSchemaBinding.enabled" -> "true") {
+      val catalog = new SessionCatalog(newBasicCatalog())
```

Comment:
to be consistent with other test cases in this file, shall we use `withBasicCatalog`?